app/assets/javascripts/components/locales/es.jsx

```diff
@@ -1,49 +1,49 @@
-  "getting_started.about_shortcuts": "If the target user is on the same domain as you, just the username will work. The same rule applies to mentioning people in statuses.",
-  "getting_started.about_developer": "The developer of this project can be followed as Gargron@mastodon.social",
-  "column.home": "Home",
-  "column.mentions": "Mentions",
-  "column.public": "Public",
-  "tabs_bar.compose": "Compose",
-  "tabs_bar.home": "Home",
-  "tabs_bar.mentions": "Mentions",
-  "tabs_bar.public": "Public",
-  "compose_form.placeholder": "What is on your mind?",
-  "compose_form.publish": "Publish",
-  "navigation_bar.settings": "Settings",
-  "navigation_bar.public_timeline": "Public timeline",
-  "navigation_bar.logout": "Logout",
-  "reply_indicator.cancel": "Cancel",
-  "search.placeholder": "Search",
-  "search.account": "Account",
-  "search.hashtag": "Hashtag",
-  "suggestions_box.who_to_follow": "Who to follow",
-  "suggestions_box.refresh": "Refresh",
-  "upload_button.label": "Add media",
-  "upload_form.undo": "Undo"
+  "column_back_button.label": "Atrás",
+  "lightbox.close": "Cerrar",
+  "loading_indicator.label": "Cargando...",
+  "status.mention": "Mencionar",
+  "status.delete": "Borrar",
+  "status.reply": "Responder",
+  "status.reblog": "Republicar",
+  "status.favourite": "Favourito",
+  "status.reblogged_by": "{name} republicado",
+  "video_player.toggle_sound": "Act/Desac. sonido",
+  "account.mention": "Mención
+  "account.edit_profile": "Editar perfil",
+  "account.unblock": "Desbloquear",
+  "account.unfollow": "Dejar de seguir",
+  "account.block": "Bloquear",
+  "account.follow": "Seguir",
+  "account.block": "Bloquear",
+  "account.posts": "Publicaciones",
+  "account.follows": "Seguir",
+  "account.followers": "Seguidores",
+  "account.follows_you": "Te sigue",
+  "getting_started.heading": "Empezando",
+  "getting_started.about_addressing": "Puedes seguir a gente si conoces su nombre de usuario y el dominio en el que están registrados, introduciendo algo similar a una dirección de correo electrónico en el formulario en la parte superior de la barra lateral.",
+  "getting_started.about_shortcuts": "Si el usuario que buscas está en el mismo dominio que tu, simplemente funcionará introduciendo el nombre de usuario. La misma regla se aplica para mencionar a usuarios.",
+  "getting_started.about_developer": "Puedes seguir al desarrollador de este proyecto en Gargron@mastodon.social",
+  "column.home": "Inicio",
+  "column.mentions": "Menciones",
+  "column.public": "Publico",
+  "tabs_bar.compose": "Redactar",
+  "tabs_bar.home": "Inicio",
+  "tabs_bar.mentions": "Menciones",
+  "tabs_bar.public": "Publico",
+  "compose_form.placeholder": "¿En qué estás pensando?",
+  "compose_form.publish": "Publicar",
+  "navigation_bar.settings": "Ajustes",
+  "navigation_bar.public_timeline": "Línea de tiempo pública",
```

Comment:
"Línea de tiempo pública" it's the text showed in my GNUsocial account (quitter.no) so I did something similar...